mlir/lib/Target/LLVMIR/ModuleTranslation.cpp

```diff
@@ -1606,6 +1606,9 @@ static void convertFunctionAttributes(LLVMFuncOp func,
     llvmFunc->addFnAttr(llvm::Attribute::NoUnwind);
   if (func.getWillReturnAttr())
     llvmFunc->addFnAttr(llvm::Attribute::WillReturn);
+  if (FramePointerKindAttr fpAttr = func.getFramePointerAttr())
```

Comment:
There is already a translation of the frame pointer attribute in `convertOneFunction` which apparently only executes for function declarations and not for definitions.

Did you see it on declarations in real inputs? If yes then let's remove the frame pointer code from `convertOneFunction`. I think this function runs on both declarations and definitions.

Reply:
> Did you see it on declarations in real inputs?

I do! Will update